src/docfx/lib/log/Error.cs

```diff
@@ -27,6 +25,16 @@ internal record Error
 
         public object?[] MessageArguments { get; init; } = Array.Empty<object?>();
 
+        public string? MsAuthor { get; init; }
```

Comment:
Consider extract a class `AdditonalErrorInfo` and put these `Ms.xyz` into that class.